en/assembly/quick_start_durandal.md

```diff
@@ -0,0 +1,202 @@
+
+
+## Mount and Orient Controller
+
+*Durandal* should be mounted on the frame positioned as close to your vehicle’s center of gravity as possible, oriented top-side up with the arrow pointing towards the front of the vehicle.
+
+![Mounting/Orientation](../../assets/flight_controller/durandal/orientation.jpg)
+
+If the controller cannot be mounted in the recommended/default orientation (e.g. due to space constraints) you will need to configure the autopilot software with the orientation that you actually used: [Flight Controller Orientation](../config/flight_controller_orientation.md).
+
+> **Tip** The board has internal vibration-isolation.
+  Do not use vibration-isolation foam to mount the controller (double sided tape is normally sufficient).
+
+
+## GPS + Compass + Buzzer + Safety Switch + LED
+
+Durandal is designed to work well with the *Pixhawk 4 GPS module*, which has an integrated compass, safety switch, buzzer and LED.
+It connects directly to the [GPS port](../flight_controller/durandal.md#gps) using the 10 pin cable.
+
+The GPS/Compass should be mounted on the frame as far away from other electronics as possible, with the direction marker towards the front of the vehicle (separating the compass from other electronics will reduce interference).
+
+![Connect compass/GPS to Durandal](../../assets/flight_controller/durandal/connection_gps_compass.jpg) 
+
+> **Note** The GPS module's integrated safety switch is enabled *by default* (when enabled, PX4 will not let you arm the vehicle).
+  To disable the safety press and hold the safety switch for 1 second.
+  You can press the safety switch again to enable safety and disarm the vehicle (this can be useful if, for whatever reason, you are unable to disarm the vehicle from your remote control or ground station).
+
+
+## Power {#power}
+
+You can use a power module or power distribution board to power motors/servos and measure power consumption.
+The recommended power modules are shown below.
+
+### PM02 v3 Power Module {#pm02_v3}
+
+The [Power Module (PM02 v3)](https://shop.holybro.com/power-modulepm02-v3_p1185.html) can be bundled with *Durandal*.
+It provides regulated power to flight controller and sends battery voltage/current to the flight controller.
+
+Connect the output of the *Power Module* as shown.
+
+![Durandal PM02v3 Power connections](../../assets/flight_controller/durandal/connection_power.jpg)
+
+
+- PM voltage/current port: connect to [POWER1](../flight_controller/durandal.md#power) port (or `POWER2`) using the 6-wire GH cable supplied.
+- PM input (XT60 male connector): connect to the LiPo battery (2~12S).
+- PM power output (XT60 female connector): wire out to any motor ESCs.
+
+> **Note** As this power module does not include power distribution wiring, you would normally just connect all the ESCs in parallel to the power module output (the ESC must be appropriate for the supplied voltage level).
```

Comment:
@bkueng FYI I updated this section to explain how you do the wiring for motors (ie resolves https://github.com/PX4/px4_user_guide/pull/609#discussion_r347674240 )